### ISSUE TITLE
Edit resource bundles in tabular view

### ISSUE THREAD

**Issue by beradrian** [essiembre/eclipse-rbe#36]
It would be nice to have some kind of tabular view, similar to a spreadsheet.
The rows will be the keys and the columns will be the locales.

This way you can check at a glance whatever is missing and edit all really in the same place.

Due to layout constraints, when displaying the tabular, the cell values can be truncated to x characters, but when you focus on a cell a bigger textarea will expand to allow editing of the entire value.

**Comment by k0ner** [comment]
What would be the advantage of having such view? As far as I understood from you proposal it could help missing some translations? Am I right? If yes then I think that much easier would be to include on-demand possibility to check if there are some missing translations.

**Comment by beradrian** [comment]
The advantage will be that you can edit all messages in all locales in the same place at once. If you want to edit related messages you can still see them all in the same view. Currently you have to switch between screens back and forth.

**Comment by teg-man** [comment]
I totally agree that a tabular view would be nice. It would massively speed up work.
The table could have the key in the first column followed by value columns - one for each language.

|.Key..................... |.German |. English.|
|............................................................|
|.table.viewer.cool..|.Ja..........|.Yes........|

Currently it is a pain switching from the key tree to the input fields. Using tab is not possible. For me an ideal version would be a table viewer with cell editors where you could use arrow and tab keys to navigate just like in Excel. Switching between mouse and keyboard when you have to edit a greater amount of entries can be really annoying.
You could even combine the current tree view with the table view using SWT's standard component [TableTree](http://www.java2s.com/Code/Java/SWT-JFace-Eclipse/DemonstratesTableTreeTableTree.htm)
